ml-agents/mlagents/trainers/ghost/trainer.py

```diff
@@ -15,7 +14,7 @@
 from mlagents.trainers.trajectory import Trajectory
 from mlagents.trainers.agent_processor import AgentManagerQueue
 
-LOGGER = logging.getLogger("mlagents.trainers")
+logger = logging.getLogger("mlagents.trainers")
```

Comment:
Why not create_logger here?

Reply:
its not clear to me where the logger is initialized. there are several import statements in each file and since learn.py is the entry point of mlagents-learn I think it makes sense to create the logger there.

Reply:
I think the only way to be absolutely sure that the basicConfig is applied is to do `import logging` and `logger = create_logger("mlagents.trainers")` at the very top of `learn.py`, since accidentally importing any file with the `getLogger` will cause the config to be ignored. 

@chriselion could we do something similar to the `tf_util.py` where we create and import the logger from logging_utils rather than the default module? It would be weird since we'd have to create the envs logger as well here.

Reply:
I think ideally we should get all the loggers going through create_logger, and they all should use the logging level decided by `options.debug`. This is probably no worse than before though, so OK to merge this today.